config/application.rb

```diff
@@ -76,24 +76,36 @@ class Application < Rails::Application
           # necessarily need replicas.
           # Docs: https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html
           raise "Elasticsearch status is #{status}" unless %w[green yellow].include?(status)
+
           "ok"
         end
 
-        ping.check :github do
-          uri = URI("https://status.github.com/api/status.json")
-          status = JSON.parse(Net::HTTP.get(uri))["status"]
-          raise "GitHub status is #{status}" unless status == "good"
+        ping.check :github, timeout: 5 do
+          status_components = JSON.parse(Net::HTTP.get(uri))["components"]
```

Comment:
Where does `uri` come from here?

Reply:
Good catch, I think it was accidentally removed from the diff.

```suggestion
          uri = URI("https://status.github.com/api/status.json")
          status_components = JSON.parse(Net::HTTP.get(uri))["components"]
```

Reply:
@d12 @talsafran Yup, missed on my `git add`! Adjusted to use the new GitHub Status API URI (`uri = URI("https://www.githubstatus.com/api/v2/components.json")`)